ui/component/publishPending/view.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react';
+import Lbry from 'lbry-redux';
+import Button from 'component/button';
+
+type Props = {
+  pending?: boolean,
+  reflectingInfo?: ReflectingUpdate,
+  claim: ?Claim,
+  checkReflecting: () => void,
+};
+
+const PublishPending = (props: Props) => {
+  const {
+    reflectingInfo = {},
+    pending,
+    // claim,
+    checkReflecting,
+  } = props;
+  // const size = claim && claim.value && claim.value.source && claim.value.source.size;
+  const { fileListItem, progress, stalled } = reflectingInfo;
+  const sdHash = fileListItem && fileListItem.sd_hash;
+  const reflecting = Object.keys(reflectingInfo).length;
+  return (
+    <span>
+      {__('Pending: %message%', {
```

Comment:
Instead of "Pending: {x}" let's just say 
`Uploading (x%)` when it's reflecting and pending confirmation or just reflecting
`Confirming` when it's awaiting confirmation